src/cljs/rems/application.cljs

```diff
@@ -646,9 +647,35 @@
     (for [event (:application/events application)]
       (update event :event/attachments (partial mapv (comp attachments-by-id :attachment/id))))))
 
-(defn- application-state [application config highlight-request-id]
+(defn- render-state-fields [application config events]
```

Comment:
I kind of preferred the older naming convention of `application-state` component instead of a "render function". Would `application-state-details` communicate that this is the details that we have initially hidden? Same logic for the others.